### PR TITLE
Feature/query cxx amsua

### PR DESCRIPTION
**Pull request by emilyhcliu**
## Description

Update YALM files for normal-feed and direct broadcast AMSU-A data based on feature/query_cxx

The related IODA ObsSpace.yaml changes can be found in feature/query_cxx branch under IODA repository

### Issue(s) addressed

Link the issues to be closed with this PR
- fixes #<[957](https://github.com/JCSDA-internal/ioda-converters/issues/957)>

### CODE REVIEW

**Comment by emilyhcliu** [comment]
Validation with GSI:
amsua_metop-c 2020110112 brightness_temperature channel1
gdas t12z 1bamua_metop-c antennaTemperature_channel1
amsua_metop-c 2020110112 brightness_temperature channel2
gdas t12z 1bamua_metop-c antennaTemperature_channel2
amsua_metop-c 2020110112 brightness_temperature channel12
gdas t12z 1bamua_metop-c antennaTemperature_channel12
amsua_metop-c 2020110112 brightness_temperature channel15
gdas t12z 1bamua_metop-c antennaTemperature_channel15

**Comment by YoulongXia-NOAA** [comment]
@emilyhcliu, I checked the yaml file, I did not find any problems. I am only confused by such a definition: heightOfStation and longname is altitude of satellite which is conflict. Not sure if it should be heightOfSatellite. I looked at the convention table, I did not find any information there. Nevertheless,  it may need to a clear definition for such a variable.

File: test/testinput/bufr_ncep_1bamua.yaml [hunk 1]
```diff
@@ -67,101 +86,92 @@ observations:
...
-          units: ""
+        - name: "MetaData/sensorChannelNumber"
+          source: variables/sensorChannelNumber
+          longName: "Sensor Channel Number"
 
         - name: "MetaData/latitude"
           source: variables/latitude
-          longName: "LATITUDE (COARSE ACCURACY)"
-          units: "degree (north)"
+          longName: "Latitude (coarse accuracy)"
```
Comment (YoulongXia-NOAA) [inline test/testinput/bufr_ncep_1bamua.yaml:113]: Remove (coarse accuracy) as it is not needed.

File: test/testinput/bufr_ncep_1bamua.yaml [hunk 2]
```diff
@@ -67,101 +86,92 @@ observations:
...
+          units: "degree_north"
           range: [-90, 90]
 
         - name: "MetaData/longitude"
           source: variables/longitude
-          longName: "LONGITUDE (COARSE ACCURACY)"
-          units: "degree (east)"
+          longName: "Longitude (coarse accuracy)"
```
Comment (YoulongXia-NOAA) [inline test/testinput/bufr_ncep_1bamua.yaml:119]: The same as above

File: test/testinput/bufr_ncep_1bamua.yaml [hunk 3]
```diff
@@ -67,101 +86,92 @@ observations:
...
-          longName: "HEIGHT OF LAND SURFACE"
+          longName: "Height of Land Surface"
           units: "m"
 
-        - name: "MetaData/altitude"
-          source: variables/altitude
-          longName: "ALTITUDE"
+        - name: "MetaData/heightOfStation"
+          source: variables/heightOfStation
+          longName: "Altitude of Satellite (above earth's surface)"
```
Comment (YoulongXia-NOAA) [inline test/testinput/bufr_ncep_1bamua.yaml:143]: Remove (above earth's surface)

**Review by nicholasesposito — COMMENTED** [review]
Looks good to me. Much like @YoulongXia-NOAA asked, I think we should remove the "high/low accuracy" and "above Earth's surface".

File: test/testinput/bufr_ncep_1bamua.yaml (hunk 1 above)
Comment (emilyhcliu) [reply test/testinput/bufr_ncep_1bamua.yaml:113]: @YoulongXia-NOAA 

The heightOfStation when applies to satellite data means the altitude of the satellite from the surface of the earth (~ 800 km for polar orbiting satellite; ~ 36,000 km for geostationary satellite)

Comment (YoulongXia-NOAA) [reply test/testinput/bufr_ncep_1bamua.yaml:113]: @emilyhcliu, can we use heightOfSatellite or altitudeOfSatellite? Here Station may confuse the users. What do you think?

**Comment by emilyhcliu** [comment]
@nicholasesposito and @YoulongXia-NOAA 
I modified YAMLs and updated the branch as you suggested.  Thanks for your reviews.

**Review by nicholasesposito — APPROVED** [review]

**Review by rmclaren — APPROVED** [review]

**Pull request merged**